VMware.CloudFoundation.PowerManagement.psm1

```diff
@@ -744,6 +744,104 @@ Function Set-VsanClusterPowerStatus {
 }
 Export-ModuleMember -Function Set-VsanClusterPowerStatus
 
+Function Invoke-VxrailClusterShutdown{
+    <#
+        .SYNOPSIS
+        Invoke shut down command on a VxRail Cluster
+
+        .DESCRIPTION
+        The cmdlet will perform a dry run test prior to initiate a shutdown command on a VxRail cluster.
+
+        .EXAMPLE
+        Invoke-VxrailClusterShutdown -server sfo-w01-vxrm.sfo.rainpole.io -user administrator@vsphere.local  -Pass VMw@re1!
+        This example powers off a Vxrail Cluster cluster which the VxRail server sfo-w01-vxrm.sfo.rainpole.io controls. 
```

Comment:
```suggestion
        This example powers off a VxRail cluster which is managed by the VxRail Manager sfo-w01-vxrm.sfo.rainpole.io. 
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108